tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__HandlerFactory.java

```diff
@@ -40,7 +40,7 @@
  *
  * @author ${author} - Initial contribution
  */
-@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.${bindingId}")
+@Component(service = ThingHandlerFactory.class, immediate = true)
```

Comment:
I thought we added the `configurationPid` in order to make the binding appear in the PaperUI binding configuration screen, or as I mistaken here?

Reply:
I will revert this change. Makes sense.

I am just asking myself why the ESH bindings do not use the `configurationPid` at the moment.

> What was it that the SAT tool complained about?

In ESH only code format and newline in `.gitignore`. But
> We included the `NOTICE` file in the `build.properties` but it doesn't exist in the archetype resources. Shall we add it? Or do we have to copy it from somewhere else? What is the suggested way to go?

In OH2 additionally
```
[ERROR] Code Analysis Tool has found: 
 4 error(s)! 
 3 warning(s) 
 4 info(s)
[ERROR] org.openhab.binding.mytestbinding.internal.MyTestBindingHandlerFactory.java:[2]
Header line doesn't match pattern ^ \* Copyright \(c\) 2010-2018 by the respective copyright holders\.$
[ERROR] org.openhab.binding.mytestbinding.internal.MyTestBindingConfiguration.java:[2]
Header line doesn't match pattern ^ \* Copyright \(c\) 2010-2018 by the respective copyright holders\.$
[ERROR] org.openhab.binding.mytestbinding.handler.MyTestBindingHandler.java:[2]
Header line doesn't match pattern ^ \* Copyright \(c\) 2010-2018 by the respective copyright holders\.$
[ERROR] org.openhab.binding.mytestbinding.MyTestBindingBindingConstants.java:[2]
Header line doesn't match pattern ^ \* Copyright \(c\) 2010-2018 by the respective copyright holders\.$
```
Steps to fix it in OH2
- replace `NOTICE` with `about.html` in `build-properties`
- run `mvn license:format`